wordpress-mobile/WordPress-iOS
# Add UI test to verify Help section loads

This PR is part of updating our login tests to account for the Unified Login & Signup flows. Context & status: paaHJt-1Kx-p2
This PR adds a new UI test,  `testSupportScreenLoads`,  to check that the Support page loads and expected items (contact support, etc) appear.

This test should catch issues like https://github.com/wordpress-mobile/WordPress-iOS/pull/16581 in the future.
So, it's important that the Support page loads, but also that the key items appear.

To test:
Run `testSupportScreenLoads` from SignupTests.swift in Xcode.

One structure question I still have: is SignupTests a good place for this test? 
I put it there because:
1. LoginTests is getting pretty busy
2. It doesn't require logging in (it tests Help from a non-logged-in state)
But I'm not sure either reason justifies putting it in SignupTests. Should it have it's own file or be elsewhere?

Please comment on implementation as well as code quality! ❤️
